recipes/jinja2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version="2.8" %}
+
+package:
+  name: jinja2
+  version: {{ version }}
+
+source:
+  fn: Jinja2-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/J/Jinja2/Jinja2-{{ version }}.tar.gz
+  md5: edb51693fe22c53cee5403775c71a99e
+
+build:
+  number:  0
+  script: python -m pip install --no-deps .
```

Comment:
If I can't remember it, I'm not a happy camper.  Sorry, I don't like having to refer back to things.  I can find them all over the place, but I don't like the 2 minute derailment to my thought process.  I'm hare-brained enough without it.